bindings/wasm/examples/worker.test.js

```diff
@@ -85,7 +85,8 @@ async function runExample(name) {
 }
 
 suite('Examples', () => {
-  test('Intro', async () => {
+  // TODO: somehow "Intro" is actually fine with vitest web-worker?
```

Comment:
I noticed this too, but it makes no sense to me. I was actually wondering if that was being skipped somehow.